css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,25 @@
+button.button {
+  color: black;
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 24px;
+  background-color: rgba(226, 215, 215, 0.865);
+}
+
+button:hover {
+  color: white;
+  background-color: rgb(16, 15, 15);
+  cursor: pointer;
+}
+
+* thead {
+  color: white;
+  background-color: black;
+}
+
+* th {
+  background-color: black;
+}
```

Comment:
I'll approve it but you're still using the `*` universal selector here. This is the same idea of using the `any` type in TypeScript and is too loose in specificity. Generally we won't want to use `*` except for our `box-sizing` in a reset file. Instead we should look at this like you want the `th` elements that are children of your `thead`. We can then go with either `thead th` (targets all `th` elements that are descendents of `thead` elements) or `thead > th` which is more specific (targets all `th` elements that are direct children of `thead` elements)